### PR TITLE
Update

### PR DESCRIPTION
**Pull request by vbrydik**

### CODE REVIEW

File: core/metrics.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
```
Comment (vbrydik) [inline core/metrics.py:1]: https://github.com/ranchlai/speaker-verification/blob/bf76570476f37cce5446f4b10b15fad9ef6baafa/metrics.py#L21

File: core/ecapa.py [hunk 2]
```diff
@@ -11,4 +11,17 @@ def __init__(self):
     
     def __call__(self, audio: Audio):
         signal, fs = torchaudio.load(audio.path)
-        return self.model.encode_batch(signal)
```
Comment (dsabodashko) [inline core/ecapa.py:14]: check the required sampling rate

File: core/pyannote.py [hunk 3]
```diff
@@ -32,8 +32,9 @@ def __call__(self, audio: Audio):
     audio1 = Audio("dataset/1-Zelenskyi/audio09.wav") 
     emb1 = model(audio1)
```
Comment (dsabodashko) [inline core/pyannote.py:33]: check the required sampling rate

File: core/titanet.py [hunk 4]
```diff
@@ -10,3 +10,16 @@ def __init__(self):
     
     def __call__(self, audio: Audio):
         return self.model.get_embedding(audio.path)
```
Comment (dsabodashko) [inline core/titanet.py:12]: check the required sampling rate

File: main.py [hunk 5]
```diff
@@ -1,90 +1,84 @@
...
+    }
+    return result
 
-    # Generate datasets for evaluation for each speaker
-    speakers_datasets = generate_speakers_dataset(speaker_to_samples_dict)
 
-    total_df = pd.DataFrame()
+def main():
+
+    dataset = make_dataset("./dataset")
```
Comment (dsabodashko) [inline main.py:60]: we need to be sure that we generate the same dataset for each experiment. I guess we can dump it into the file at the first run and load it for rest of experiments

File: main.py [hunk 6]
```diff
@@ -1,90 +1,84 @@
...
 
-def main():
-    # Get speaker to samples map
-    dataset_dir = "./dataset"
-    speakers = glob.glob(os.path.join(dataset_dir, "*"))
-    speakers = speakers[:2]
-    speaker_to_samples_dict = { s: glob.glob(os.path.join(s, "*.wav")) for s in speakers }
+    ee_rate, thresh, fa_rate, fr_rate = compute_eer(scores, labels)
+    min_dcf = compute_min_dcf(fr_rate, fa_rate)
+    fa_score, fr_score = compute_far_frr(scores, labels, thresh)
```
Comment (dsabodashko) [inline main.py:45]: chack if `ee_rate == (fa_score + fr_score) / 2`

**Review by dsabodashko — APPROVED** [review]
@vbrydik Nice work!

**Pull request merged**